x11-misc/xpad/xpad-4.2.ebuild

```diff
@@ -1,11 +1,11 @@
-# Copyright 1999-2014 Gentoo Foundation
+# Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=5
 inherit eutils
```

Comment:
what is this needed for? also check whether `DOCS` is really needed, as EAPI 5 already has the default `DOCS` set in `src_install`